src/main/cc/wfa/virtual_people/common/field_filter/utils/type_convert_util_test.cc

```diff
@@ -0,0 +1,141 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "wfa/virtual_people/common/field_filter/utils/type_convert_util.h"
+
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "gmock/gmock.h"
+#include "google/protobuf/descriptor.h"
+#include "google/protobuf/message.h"
+#include "gtest/gtest.h"
+#include "src/main/cc/wfa/virtual_people/common/field_filter/test/test.pb.h"
+
+namespace wfa_virtual_people {
+namespace {
+
+using ::wfa_virtual_people::test::TestProtoB;
+
+TEST(ConvertToNumericTest, TestInt32) {
+  auto output = ConvertToNumeric<int32_t>("1");
+  EXPECT_TRUE(output.ok());
+  EXPECT_EQ(output.value(), 1);
```

Comment:
`EXPECT_THAT(output, IsOkAndHolds(1));`

https://github.com/world-federation-of-advertisers/any-sketch/blob/main/src/test/cc/testutil/matchers.h

Reply:
Done.